src/sentry/api/endpoints/relay_index.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import
+
+from rest_framework.permissions import IsAuthenticated
+
+from sentry.api.base import Endpoint
+from sentry.api.paginator import OffsetPaginator
+from sentry.api.serializers import serialize
+from sentry.models import Relay
+
+
+class RelayIndexEndpoint(Endpoint):
+    permission_classes = (IsAuthenticated, )
+
+    def get(self, request):
```

Comment:
Without reading much, this seems bad that we'd just return all relays to anyone logged in. I feel like we'd want this behind superuser or something.

Reply:
Yeah. We should change that probably before deploy.